src/App.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import ClassNames from 'classnames';
```

Comment:
```suggestion
import cn from 'classnames';
```
зазвичай так скорочують